apache/superset
# feat(native-filters): hide filterBar and toggle icon when in editMode

### SUMMARY
sometimes I can do frontend.
resolves #12629
### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

https://user-images.githubusercontent.com/47772523/107872943-e92bd800-6eb6-11eb-95f9-d585455ae9be.mov





### TEST PLAN
1. enable DASHBOARD_NATIVE_FILTERS FF
2. go to a dashboard 
3. toggle the filter bar
4. click edit dashboard icon
5. check filter bar disappears and toggle icon also 

### ADDITIONAL INFORMATION


- [X] Has associated issue: #12629
